config/grad.uiowa.edu/views.view.thesis_defenses.yml

```diff
@@ -254,6 +255,7 @@ display:
             person: '#63666a'
             scholar: '#63666a'
             scholarship: '#63666a'
+            student_profile: '#3a87ad'
```

Comment:
I didn't change this.

Reply:
Probably the `student_profile` content type was added after the view was last exported. I updated it to the same color setting as the others.